src/pages/studyView/studyPageHeader/rightPanel/RightPanel.tsx

```diff
@@ -53,16 +57,88 @@ export default class RightPanel extends React.Component<IRightPanelProps, {}> {
         );
     }
 
+    @computed get isQueryLimitReached(): boolean {
```

Comment:
did we just not check for this in study view query in past?

Reply:
if we did I couldn't find it